lib/services/batchManagement/README.md

```diff
@@ -11,25 +18,24 @@ npm install azure-arm-batch
 
 ## How to use
 
-### Authentication, client creation and listing accounts as an example
-
- ```javascript
- const msRestAzure = require('ms-rest-azure');
- const batchManagementClient = require('azure-arm-batch');
-
- // Interactive Login
- // It provides a url and code that needs to be copied and pasted in a browser and authenticated over there. If successful, 
- // the user will get a DeviceTokenCredentials object.
- msRestAzure.interactiveLogin().then((credentials) => {
-  let client = new batchManagementClient(credentials, 'your-subscription-id');
-  return client.batchAccountOperations.list(rgName);
- }).then((result) => {
-  console.dir(result, {depth: null, colors: true});
-}).catch ((err) => {
-  console.log('An error occurred.');
+### Authentication, client creation and get batchAccount as an example.
+
+```javascript
+const msRestAzure = require("ms-rest-azure");
+const BatchManagementClient = require("azure-arm-batch");
+msRestAzure.interactiveLogin().then((creds) => {
+    const subscriptionId = "<Subscription_Id>";
+    const client = new BatchManagementClient(creds, subscriptionId);
+    const resourceGroupName = "testresourceGroupName";
+    const accountName = "testaccountName";
+    return client.batchAccount.get(resourceGroupName, accountName).then((result) => {
+      console.log("The result is:");
+      console.log(result);
+    });
+}).catch((err) => {
+  console.log('An error ocurred:');
```

Comment:
How'd this happen?

Reply:
Oops! Typo. I've already put out a PR to fix it in the generator.